test/support/integration_case.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Livebook.EnterpriseIntegrationCase do
+  use ExUnit.CaseTemplate
+
+  alias LivebookTest.Integration.EnterpriseServer
+
+  using do
+    quote do
+      use LivebookWeb.ConnCase
+
+      @moduletag :enterprise_integration
+
+      alias LivebookTest.Integration.EnterpriseServer
+    end
+  end
+
+  setup_all do
+    EnterpriseServer.start_link()
```

Comment:
I think we should do `GenServer.start`, otherwise it will be terminated with the test process. This way it either starts or returns `{:already_started, pid}`, which we expect. @josevalim unless there's a more appropriate way :)

Reply:
with `GenServer.start` it fails only one test and only passes if I make this change:

`test/livebook_web/live/hub/new/enterprise_component_test.exs`:

```diff
-      assert {:ok, view, _html} =
-               view
-               |> element("#enterprise-form")
-               |> render_submit(%{"enterprise" => attrs})
-               |> follow_redirect(conn)
+      res =
+        view
+        |> element("#enterprise-form")
+        |> render_submit(%{"enterprise" => attrs})
+
+      assert {:ok, view, _html} = follow_redirect(res, conn)
```

Reply:
Sounds odd to make this change, perhaps it's an async issue?

Reply:
I thought it was an intermittent test, but isn't. This test always fail

```
  1) test enterprise persists new hub (LivebookWeb.Hub.New.EnterpriseComponentTest)
     test/livebook_web/live/hub/new/enterprise_component_test.exs:13
     ** (RuntimeError) LiveView did not redirect
     code: |> follow_redirect(conn)
     stacktrace:
       test/livebook_web/live/hub/new/enterprise_component_test.exs:52: (test)
```

Reply:
@josevalim can source change affect ExUnit execution order?

@aleDsz maybe try printing the LV content to see if the page has any errors, or any information?

Reply:
What if you run just that specific test, does it still fail then?

Reply:
Doesn't fail, probably other tests are conflicting with this one

Reply:
Maybe it's because the hub is already created when we try to create it?

Reply:
That's weird, If I run the `mix test.enterprise test/livebook_web/live/hub` it fails, but If I run `mix test.enterprise test/livebook_web/live/hub/new` it doesn't.

Since the `edit_live_test.exs` uses random ID for enterprise, It shouldn't fail tbh. I'm kinda lost here

Reply:
You can also try IO.inspect-ing in LV, without changing the test.

Reply:
I can't belive that, IO.inspect-ing the LV did make it fail hahaha
And removing the IO.inspect made it work... Perhaps compile state bug or smth?

Reply:
Well, in the end, it works